docs/production/install.md

```diff
@@ -4,7 +4,7 @@ Make sure you want to install a Zulip production server. If you'd
 instead like to test or develop a new feature, we recommend the
 [Zulip server development environment](../development/overview.html#requirements) instead.
 
-You'll need an Ubuntu system that satisfies
+You'll need an Ubuntu or Debian system that satisfies
```

Comment:
I dropped this junk, just because I think documenting this for production will require also changing `production/requirements` and maybe several more places; would grep for `Xenial`

Reply:
Oops, that was a typo meant to say "dropped this hunk".  Sorry for the what must have sounded pretty disrespectful :(.

Reply:
Hahaha no worries! I imagined it was a typo, and it actually made me laugh :laughing:.